### ISSUE TITLE
OCR addon does not work in Windows 7 anymore when using NVDA 2018.2

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda-ocr#3]
Dear NV Access Team,

there are still many users of Windows 7 complaining that the OCR addon does not work anymore, especially when using NVDA 2018.2.

Is there any update in process for this addon?

Comments are appreciated.

**Comment by Adriani90** [comment]
CC: @michaelDCurran, @feerrenrut, @jsteh

**Comment by lukaszgo1** [comment]
@Adriani90 I am using OCR add-on on Windows 7 successfully almost everyday. Can you please elaborate on what you mean by not working?

**Comment by feerrenrut** [comment]
This seems quite out of date and without much information. Closing the issue.
